keps/sig-release/3720-freezing-k8s-gcr-io/README.md

```diff
@@ -0,0 +1,158 @@
+
+There was a plan to redirect `k8s.gcr.io` to `registry.k8s.io` but it didn't [work out](https://kubernetes.slack.com/archives/CCK68P2Q2/p1666725317568709) so we backported the image registry defaults to 1.25, 1.24, 1.23, and 1.22 so all the patch releases from December 2022 will using the new registry by default.
+
+We are currently exceeding our budget as it will take quite a while for end users to upgrade Kubernetes to v1.25 so we want to incentivise our end users to move to the new registry as fast as possible by freezing the registry by 1.27. This would mean that all subsequent image releases will not be available on the old registry.
+
+### Goals
+
+Freeze `k8s.gcr.io` image registry and push all new images and tags exclusively to the `registry.k8s.io` image registry.
+
+### Non-Goals
+
+- `registry.k8s.io` internal implementations details. That is handled separately by sig-k8s-infra.
+
+## Proposal
+
+Freeze the `k8s.gcr.io` image by not pushing any new digests or tags after 1.27 release. The 1.27 release itself won't be available on `k8s.gcr.io`.
+
+### Risks and Mitigations
+
+There are no risks. The old registry will still be available and you can pull the images before 1.27 on there. This change will also
+affect other users of k8s.gcr.io who should have already updated their helm charts and manifests to use the new registry.
+
+## Design Details
+
+The image promotion process is described [here](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io). Please read it for full details.
+
+This is the planned technical changes(grouped by repos):
+
+- k-sigs/promo-tools
+  - merge https://github.com/kubernetes-sigs/promo-tools/pull/669
+- k/k8s.io
+  - Fix https://github.com/kubernetes/k8s.io/issues/4611
+  - clean up the contents of the `registry.k8s.io` folder. Most of the content should be in k/registry.k8s.io repository
+  - duplicate the top level folder `k8s.gcr.io` in the repo and call it `registry.k8s.io`
+- k/test-infra
+  - blockade the k8s.gcr.io folder in k/k8s.io repository. blockade is a prow plugin that rejects PRs that modify specific folders/files.
+  - update the ProwJobs [post-k8sio-image-promo](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml) and [pull-k8sio-cip](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/releng/artifact-promotion-presubmits.yaml) `thin-manifest-dir` flags to point to the new folder
+
+
+### Test Plan
+
+This is not applicable.
+
+### Graduation Criteria
+
+This is not applicable.
+
+### Upgrade / Downgrade Strategy
+
+When users upgrade to various kubernetes versions that use the new image registry, they will be able to pull images from the new
+registry.
+
+### Version Skew Strategy
+
+This is not applicable.
+
+## Communication Plan
+
+This is a major change and requires an appropriate communication plan.
```

Comment:
We should probably enumerate more about what the communication plan is, or at least some high-level avenues for communicating this change? I think this requires more than just an email to k-dev.